src/with_logabsdet_jacobian.jl

```diff
@@ -0,0 +1,43 @@
+function ChangesOfVariables.with_logabsdet_jacobian(::typeof(log1pexp), x::Real)
+    y = log1pexp(x)
+    return y, x - y
+end
+
+function ChangesOfVariables.with_logabsdet_jacobian(::typeof(logexpm1), x::Real)
+    y = logexpm1(x)
+    return y, (x - y)
```

Comment:
Are the brackets needed? You didn't use them above:
```suggestion
    return y, x - y
```